valhalla/baldr/traffictile.h

```diff
@@ -0,0 +1,129 @@
+#ifndef VALHALLA_BALDR_TRAFFICTILE_H_
```

Comment:
nit: `#pragma once`

Reply:
I'm aware of `#pragma once`, but the general style across (almost) all other headers is to use include guards.  I went with consistency over brevity.  If `#pragma once` is to be the preferred include guard mechanism, we should convert everything to use it.

Reply:
yeah ive been slowly changing them as i touch files. would be easy enough to do a single pr though